src/otx/recipe/classification/h_label_cls/efficientnet_b0.yaml

```diff
@@ -7,6 +7,8 @@ model:
       class_path: torch.optim.SGD
       init_args:
         lr: 0.0049
+        momentum: 0.9
+        weight_decay: 0.0001
```

Comment:
efficientnet-b0 is not timm model, Is this the right change?

Reply:
I updated the name of PR. Thank you for letting me know this.